index.html

```diff
@@ -6,174 +6,186 @@
-    <div class="projectinfo3">
-      <h3 class="projectname">Project name goes here</h3>
-      <ul class="languages">
-        <li class="html">HTML/CSS</li>
-        <li class="ruby">Ruby on Rails</li>
-        <li class="js">JavaScript</li>
-      </ul>
-      <button type="button" class="buttonproject">See this Project<img src="images/arrow.png" alt="ARROW"></button>
-    </div>
-  </div>
+ <div class="description second"> 
+   <img src="images/proj2.png" alt="Web Designing">
+   <div class="projectinfo2">
+     <h3 class="projectname">Project name goes here</h3>
+     <ul class="languages">
+       <li class="html">HTML/CSS</li>
+       <li class="ruby">Ruby on Rails</li>
+       <li class="js">JavaScript</li>
+     </ul>
+     <button type="button" class="buttonproject">See this Project<img src="images/arrow.png" alt="ARROW"></button>
+   </div>
+ </div>
+ <div class="description third"> 
+   <img src="images/proj3.png" alt="E-commerce website">
+   <div class="projectinfo3">
+     <h3 class="projectname">Project name goes here</h3>
+     <ul class="languages">
+       <li class="html">HTML/CSS</li>
+       <li class="ruby">Ruby on Rails</li>
+       <li class="js">JavaScript</li>
+     </ul>
+     <button type="button" class="buttonproject">See this Project<img src="images/arrow.png" alt="ARROW"></button>
+   </div>
+ </div>
+
+ <div class="description fourth"> 
+   <img src="images/proj4.png" alt="Marketing">
+   <div class="projectinfo4">
+     <h3 class="projectname">Project name goes here</h3>
+     <ul class="languages">
+       <li class="html">HTML/CSS</li>
+       <li class="ruby">Ruby on Rails</li>
+       <li class="js">JavaScript</li>
+     </ul>
+     <button type="button" class="buttonproject">See this Project<img src="images/arrow.png" alt="ARROW"></button>
+   </div>
+ </div>
+
+ <div class="description fifth"> 
+   <img src="images/proj5.png" alt="Agency">
+   <div class="projectinfo5">
+     <h3 class="projectname">Project name goes here</h3>
+     <ul class="languages">
+       <li class="html">HTML/CSS</li>
+       <li class="ruby">Ruby on Rails</li>
+       <li class="js">JavaScript</li>
+     </ul>
+     <button type="button" class="buttonproject">See this Project<img src="images/arrow.png" alt="ARROW"></button>
+   </div>
+ </div>
+ <div class="description sixth">
+   <img src="images/proj6.png" alt="PPC">
+   <div class="projectinfo6">
+     <h3 class="projectname">Project name goes here</h3>
+     <ul class="languages">
+       <li class="html">HTML/CSS</li>
+       <li class="ruby">Ruby on Rails</li>
+       <li class="js">JavaScript</li>
+     </ul>
+     <button type="button" class="buttonproject">See this Project<img src="images/arrow.png" alt="ARROW"></button>
+   </div>
+ </div>
+ <div class="background1"></div>
+ <div class="background2"></div>
+ <div class="background3"></div>
+</section>
+
+<!-- Window Popup JavaScript -->
+<section class="popup-window" id="popup-window">
+ <div class="closebtn">
```

Comment:
- [ ] On desktop screens When I open the model and close it, the page is still showing the background I need to refresh the page to get back to the normal state of the page. Please fix this issue.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/163249813-6bac8d4a-00f0-4bf7-a137-a53e2d61f611.png)